electron/gog/setup.ts

```diff
@@ -217,6 +270,7 @@ async function setup(appName: string): Promise<void> {
             }
         ],
     */
+    //TODO
```

Comment:
What needs to be done here ?

Reply:
Execute instruction based on example above.